Source/WebCore/Modules/webauthn/AuthenticatorAttestationResponse.cpp

```diff
@@ -36,6 +36,12 @@
 
 namespace WebCore {
 
+#if HAVE(SWIFT_CPP_INTEROP)
+static constexpr auto useCryptoKit = UseCryptoKit::Yes;
+#else
+static constexpr auto useCryptoKit = UseCryptoKit::No;
+#endif
```

Comment:
I think you should refactor the WebCore interfaces that take a `useCryptoKit` parameter: remove the parameter entirely, and use `#if` conditions inside each function to decide whether to call into CryptoKit or CommonCrypto.

As is, the code suggests that it's possible for to call into CC on modern platforms by passing the wrong parameter.

Reply:
it turns out.. All the APIs already have the same check inside them so we don't need the `#if HAVE(SWIFT_CPP_INTEROP)` here and I have removed it..